ouster-lidar/ouster-ros
# ROS-350[NOETIC]: 't' timestamp field content is not plausible

## Related Issues & PRs
- resolves #350

## Summary of Changes
- Don't apply offset to destaggered timestamps within the point cloud
- Make sure the the timestamps align correctly when destaggerd is option is OFF
- Misc: update bug/question templates

## Validation
* run the driver with default option (destaggered=ON, organized=ON)
  -  observe that the timestamps are consistent
* run the driver with default option (destaggered=OFF, organized=ON)
  - observe that the timestamps appear staggerd
